common/services/prismic/types.js

```diff
@@ -69,3 +69,11 @@ export type PaginatedResults<T> = {|
 |}
 
 export type DocumentType = 'articles' | 'webcomics' | 'events' | 'exhibitions';
+
+export type Team = {|
+  id: string,
+  title: string,
+  email: string,
+  phone: string,
+  url: string
+|}
```

Comment:
Could we put this in `model/teams.js`?
These are for the Prismic service types.